app/src/main/java/gov/hhs/cdc/trustedintermediary/auth/AuthRequestValidator.java

```diff
@@ -21,10 +21,12 @@ public class AuthRequestValidator {
     private static final AuthRequestValidator INSTANCE = new AuthRequestValidator();
 
     @Inject private AuthEngine jwtEngine;
-    @Inject private Cache keyCache;
+    @Inject Cache keyCache;
```

Comment:
Seems this is the only inject we actually use in this class. Would a small doc be good?

Reply:
Are you sure?  I see `jwtEngine`, `secrets`, and `logger` being used below.

Reply:
I added some extra comments on the class. I think all the injects are used unless I'm misunderstanding what you mean.